packages/lab/src/__tests__/__e2e__/file-drop-zone/FileDropZone.cy.tsx

```diff
@@ -0,0 +1,61 @@
+import { composeStories } from "@storybook/react";
+import * as fileDropZoneStories from "@stories/file-drop-zone/file-drop-zone.stories";
+import { checkAccessibility } from "../../../../../../cypress/tests/checkAccessibility";
+
+const composedStories = composeStories(fileDropZoneStories);
+const { Default } = composedStories;
+
+describe("Given a file drop zone", () => {
+  checkAccessibility(composedStories);
+
+  it("should render an input that accepts multiple files", () => {
+    cy.mount(<Default />);
+    cy.findByTestId("file-input").should("have.attr", "type", "file");
+    cy.findByTestId("file-input").should("have.attr", "multiple");
+  });
+  it("should accept files on drop", () => {
+    cy.mount(<Default />);
+    cy.findByTestId("file-drop-zone-example").selectFile(
+      {
+        contents: Cypress.Buffer.from("file"),
+        fileName: "image",
+        mimeType: "image/jpg",
+      },
+      {
+        action: "drag-drop",
+      }
+    );
+    cy.findByTestId("file-drop-zone-example").should(
+      "have.class",
+      "saltFileDropZone-success"
+    );
+  });
+  it("should trigger onDrop when files are dropped", () => {
+    const dropFilesSpy = cy.stub().as("dropFilesSpy");
```

Comment:
```suggestion
    const dropSpy = cy.stub().as("dropSpy");
```
I prefer these to match the callback name